lib/translation_io/config.rb

```diff
@@ -6,9 +6,14 @@ class Config
     attr_accessor :verbose
     attr_accessor :test
 
+    attr_accessor :multi_domain, :domain_names, :domain_api_keys
+    attr_accessor :domain_source_locales, :domain_target_locales, :domain_folders
+    attr_accessor :text_domain
```

Comment:
Add new keys needed to enable multi-domain support.
`multi_domain` is a boolean flag to eanble the new code
`domain_names` is an array of strings corresponding to the name of each domain
`domain_api_keys` is an array of strings corresponding to the api_key of each domain
`domain_source_locales` is an array of strings for the source locales of each domain
`domain_target_locales` is an array of arrays of strings corresponding to the target locales of each domain
`domain_folders` is an array of arrays of string paths to folders.  
- This enables the configuration of certain folders/subfolders of the site for each domain
- an empty list is used to indicate the 'catchall' or 'default' domain